### PR TITLE
Adds safeguard to modification of data files and edits UG DG 

### PR DESCRIPTION
**Pull request by kangyeelim**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -1140,6 +1140,17 @@ Data in treasurerPro is saved automatically after executing any command. There i
 *Q*: Can I modify the data files myself? +
 *A*: No, any external modification to the app's data files *WILL* result in loss of all data. You have been warned.
 
+*Q*: What happens if I accidentally modified the data files? +
+*A*: If the transactionHistory.txt file is changed, the Home and Reimbursements tab will start with an empty table.
+If the inventoryInformation.txt file is changed, the app will not open. To remedy, delete the txt file.
+If the mytasks.txt and tasks.txt files are changed, the Overview tab lose the notification reminders set.
```
Comment (dawo5010) [inline docs/UserGuide.adoc]: Should be overviewInformation.txt

Comment (kangyeelim) [reply docs/UserGuide.adoc]: alright will change in UG DG

Comment (dawo5010) [reply docs/UserGuide.adoc]: thank you!

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -1143,7 +1143,7 @@ Data in treasurerPro is saved automatically after executing any command. There i
 *Q*: What happens if I accidentally modified the data files? +
 *A*: If the transactionHistory.txt file is changed, the Home and Reimbursements tab will start with an empty table.
 If the inventoryInformation.txt file is changed, the app will not open. To remedy, delete the txt file.
-If the mytasks.txt and tasks.txt files are changed, the Overview tab lose the notification reminders set.
+If the overviewInformation.txt files are changed, the Overview tab lose the notification reminders set.
```
Comment (dawo5010) [inline docs/UserGuide.adoc]: Sorry to nitpick but singular 'file' HAHA

**Pull request merged**